integration/system/options_test.go

```diff
@@ -0,0 +1,118 @@
+package system // import "github.com/docker/docker/integration/system"
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"runtime"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/docker/docker/api"
+	"github.com/docker/docker/api/types"
+	req "github.com/docker/docker/integration-cli/request"
+	"github.com/gotestyourself/gotestyourself/skip"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestOptionsRoute(t *testing.T) {
+	resp, _, err := req.Do("/", req.Method(http.MethodOptions))
+	require.NoError(t, err)
+	assert.Equal(t, resp.StatusCode, http.StatusOK)
+}
+
+func TestGetEnabledCORS(t *testing.T) {
+	resp, body, err := req.Get("/version")
+	require.NoError(t, err)
+	defer body.Close()
+	assert.Equal(t, resp.StatusCode, http.StatusOK)
+	// TODO: @runcom incomplete tests, why old integration tests had this headers
+	// and here none of the headers below are in the response?
```

Comment:
This test doesn't look to be useful anymore. It's been disabled for so long now I think we can aprobably remove it as the original intent is probably lost.